components/mercury/mercury.app.js

```diff
@@ -0,0 +1,40 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "mercury",
+  methods: {
+    _getBaseURL() {
+      return "https://backend.mercury.com/api/v1";
+    },
+    _getHeaders() {
+      return {
+        Authorization: `Bearer ${this.$auth.api_key}`,
+        "Content-Type": "application/json",
+      };
+    },
+    async _makeGetRequest(endpoint, params = null) {
```

Comment:
Can you abstract this method to accept the HTTP method and rename `_makeRequest`? That way it'll support `POST` requests, etc. naturally as we evolve the app file.